src/bundle/Resources/public/js/scripts/fieldType/ezkeyword.js

```diff
@@ -0,0 +1,70 @@
+(function (global) {
+    const SELECTOR_FIELD = '.ez-field-edit--ezkeyword';
+    const SELECTOR_LABEL_WRAPPER = '.ez-field-edit__label-wrapper';
+
+    class EzKeywordValidator extends global.eZ.BaseFieldValidator {
+        validateKeywords(event) {
+            const isRequired = event.target.required;
+            const isEmpty = !event.target.value.trim().length;
+            const isError = (isEmpty && isRequired);
+            const label = event.target.closest(SELECTOR_FIELD).querySelector('.ez-field-edit__label').innerHTML;
+            const result = { isError };
+
+            if (isError) {
+                result.errorMessage = global.eZ.errors.emptyField.replace('{fieldName}', label);
+            }
+
+            return result;
+        }
+    }
+
+    const updateValue = (input, event) => {
+        input.value = event.detail.tags.map(tag => tag.label).join();
```

Comment:
Is it correct? I think it will join tags `Platform` and `Studio` to `PlatformStudio` or the `,` is part of the tag (I assume backend wants comma separated value)?

Reply:
The `join()` function adds a coma by default if no delimiter is set, so there's no reason to add it as a param.